src/components/FuelUsed/FuelForm.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+export default function FuelForm () {
+    return (
+        <div>
```

Comment:
Looks like we're missing some Tailwind styling. Everything is in one row. I would suggest some styling at the minimum for now, such as className='flex flex-col justify-center'

Reply:
This is just a placeholer page while I was learning things. I scrapped this from start.

Reply:
I'm not sure why it showed in this PR because I've reset this component entirely.

Reply:
Did you push your changes?